src/bin/pregen.rs

```diff
@@ -22,37 +22,37 @@ fn n_groups<T>(mut values: Vec<T>, n: usize) -> Vec<Vec<T>> {
     groups
 }
 
-fn find_patterns(targets: Vec<i64>) -> BTreeMap<i64, (String, String)> {
+fn find_patterns(targets: Vec<i64>) -> BTreeMap<i64, String> {
     let mut data = BTreeMap::new();
     let re = Regex::new(r"^aqaa").unwrap();
 
     for (i, &target) in targets.iter().enumerate() {
         println!("{}/{}", i + 1, targets.len());
 
-        let GeneratedNumber { direction, pattern, .. } =
+        let GeneratedNumber { pattern, .. } =
             generate_number_pattern(target.into(), false, false, hexnumgen::GeneratorOptions::AStar(AStarOptions {}))
                 .unwrap();
 
-        if !target.is_zero() {
-            let negative_pattern = re.replace(&pattern, "dedd").to_string();
-            data.insert(-target, (Direction::NorthEast.to_string(), negative_pattern));
-        }
-
-        data.insert(target, (direction, pattern));
+        data.insert(target, re.replace(&pattern, "").to_string());
     }
 
     data
 }
 
-fn worker(targets: Vec<i64>, tx: Sender<BTreeMap<i64, (String, String)>>) {
+fn worker(targets: Vec<i64>, tx: Sender<BTreeMap<i64, String>>) {
     tx.send(find_patterns(targets)).unwrap();
 }
 
 #[derive(Parser)]
+#[command(author = "Gamma Delta", version = "1", about = "generates hexcasting number literals")]

```

Comment:
Would prefer using the syntax to automatically pull these from Cargo.toml. Though honestly I don't think this line is really necessary at all.

Reply:
does clap have that option?

Reply:
yes: https://docs.rs/clap/latest/clap/_derive/_tutorial/index.html#configuring-the-parser